src/libsync/account.cpp

```diff
@@ -179,6 +179,21 @@ QString Account::displayName() const
     return displayName;
 }
 
+QString Account::shortcutName() const
+{
+    auto url_part = QStringLiteral(_url.host());
```

Comment:
this is failing to compile
```suggestion
    auto url_part = QString(_url.host());
```
might be the solution

Reply:
> this is failing to compile
> 
> might be the solution

I have applied these changes.
Building with the github runner works, but building with my selfhosted runner does not.